data/gloperate/shaders/lightProcessing.glsl

```diff
@@ -0,0 +1,54 @@
+#ifndef LIGHT_PROCESSING
+#define LIGHT_PROCESSING
+
+#define LIGHT_TYPE_AMBIENT 1
+#define LIGHT_TYPE_DIRECTIONAL 2
+#define LIGHT_TYPE_POINT 3
+#define LIGHT_TYPE_POINT_ATT 4
+
+#ifdef LIGHT_PROCESSING_DIFFUSE
+    #include </lightProcessingDiffuse.glsl>
+#endif
+
+#ifdef LIGHT_PROCESSING_PHONG
```

Comment:
Provide a fallback if neither of the two constants are defined (default to diffuse?)